ts/components/SectionStatus/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { IOColors } from "../core/variables/IOColors";
+import StatusContent from "./StatusContent";
+
+type OwnProps = {
+  sectionKey: SectionStatusKey;
+  onSectionRef?: (ref: React.RefObject<View>) => void;
+};
+
+type Props = OwnProps & ReturnType<typeof mapStateToProps>;
+
+export const statusColorMap: Record<LevelEnum, string> = {
+  [LevelEnum.normal]: IOColors.aqua,
+  [LevelEnum.critical]: IOColors.red,
+  [LevelEnum.warning]: IOColors.orange
+};
+
+const statusIconMap: Record<LevelEnum, string> = {
+  [LevelEnum.normal]: "io-complete",
+  [LevelEnum.critical]: "io-warning",
+  [LevelEnum.warning]: "io-info"
+};
+const color = IOColors.white;
+
+/**
+ * this component shows a full width banner with an icon and text
+ * it could be tappable if the web url is defined
+ * it renders nothing if for the given props.sectionKey there is no data or it is not visible
+ * @param props
+ */
+const SectionStatus: React.FC<Props> = (props: Props) => {
+  if (props.sectionStatus === undefined) {
+    return null;
+  }
+  if (props.sectionStatus.is_visible !== true) {
+    return null;
+  }
+
+  const viewRef = React.createRef<View>();
+  const sectionStatus = props.sectionStatus;
+  const iconName = statusIconMap[sectionStatus.level];
+  const backgroundColor = statusColorMap[sectionStatus.level];
+  const locale = getFullLocale();
+  const maybeWebUrl = maybeNotNullyString(
+    sectionStatus.web_url && sectionStatus.web_url[locale]
+  );
+  const navigation = useNavigationContext();
+
+  const handleOnSectionRef = () => {
+    if (viewRef.current) {
+      props.onSectionRef?.(viewRef);
+    }
+  };
+
+  const accessibilityLabel = maybeWebUrl.fold(
+    `${sectionStatus.message[locale]}, ${I18n.t("global.accessibility.alert")}`,
+    _ =>
+      `${sectionStatus.message[locale]}, ${I18n.t(
+        "global.sectionStatus.moreInfo"
+      )}`
+  );
+
+  const innerSectionChildren = maybeWebUrl.fold(
+    <>{sectionStatus.message[locale]}</>,
+    _ => (
+      <>
+        {`${sectionStatus.message[locale]} `}
+        <Text
+          testID={"SectionStatusComponentMoreInfo"}
+          style={{
+            color,
+            textDecorationLine: "underline",
+            fontWeight: "bold"
+          }}
+        >
+          {I18n.t("global.sectionStatus.moreInfo")}
+        </Text>
+      </>
+    )
+  );
+
+  React.useEffect(() => {
+    handleOnSectionRef();
+    const unsubscribe = navigation?.addListener("didFocus", handleOnSectionRef);
+    return () => unsubscribe?.remove();
+  }, [viewRef]);
+
+  return maybeWebUrl.fold(
+    <StatusContent
+      accessibilityLabel={accessibilityLabel}
+      backgroundColor={backgroundColor}
+      iconColor={color}
+      iconName={iconName}
+      testID={"SectionStatusComponentContent"}
+      viewRef={viewRef}
+    >
+      {innerSectionChildren}
+    </StatusContent>,
+    webUrl => (
+      <Pressable
+        testID={"SectionStatusComponentPressable"}
```

Comment:
Talking about accessibility, this item won't be correctly read by VO, maybe we should add a role and perhaps a specific hint too to make let user understand that activating the item the browser would be opened

what do you think?

@thisisjp maybe you can help us here 😄

Reply:
good catch! we should add, after "link": "opens in your browser app" (italiano: "si apre nel tuo browser")

Reply:
Thank you @CrisTofani ! You helped me catch a sneaky edge case 🎉 
The `Pressable` item doesn't render a new node but merges down with the child and adds touch capabilities: that's why I assumed the properties on `StatusContent` below were always working. It turns out they don't in the `messages` screen, where the banner is on top. No idea why 🤯  yet applying them to the `Pressable` element fixed it!

wrt the _hint_: I'm gonna use the existing [global.accessibility.linkHint](https://github.com/pagopa/io-app/blob/master/locales/it/index.yml#L79).

I've addressed this comment and reworked a bit the code in [this commit](https://github.com/pagopa/io-app/pull/3208/commits/1be5dfd327e31616b4b6f8c0f8cd54c4d3fd4361).

Please find a screenshot below showing the a11y properties:

![Screenshot 2021-08-06 at 10 35 45](https://user-images.githubusercontent.com/2094604/128482691-1c4c8b5a-3725-46c5-9d04-bf44c5073ce4.png)